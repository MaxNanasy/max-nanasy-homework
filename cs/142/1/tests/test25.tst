main() {	print((x * 5));}