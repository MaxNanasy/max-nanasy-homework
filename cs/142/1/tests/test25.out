 MAIN  (  )  {  PRINT  (  (  ident  *  number  )  )  ;  }  <EOT> 